Sources/Tracking/Service/Request/TrackRequestBody.swift

```diff
@@ -12,3 +12,9 @@ enum EventType: String, Codable {
     case event
     case screen
 }
+
+// TODO: Is this the right place?
+public struct TrackEventTypeForAnalytics: Codable {
```

Comment:
what is this event for?

Reply:
This struct is used [here](https://github.com/customerio/customerio-ios/blob/dd054e015ba2a0f1462a1d3e9eb705e4fa6dccde/Sources/Tracking/CustomerIOImplementation.swift#L356) to convert the attributed string data to json. Since we do not have a data model to cater this so I created a new one here. Does it help ?